sauron/sauron.py

```diff
@@ -38,14 +36,33 @@ def fetch(url):
 
     return session.get(url)
 
-
 @plugin.init()
-def init(plugin, options, configuration, **kwargs):
-    plugin.api_endpoint = options["sauron-api-endpoint"]
+def init(plugin, options, **kwargs):
+    plugin.api_endpoint = options.get("sauron-api-endpoint", None)
+    plugin.log("plugin.api_endpoint = %s" % plugin.api_endpoint)
+    
     if not plugin.api_endpoint:
         raise SauronError("You need to specify the sauron-api-endpoint option.")
         sys.exit(1)
 
+    # Test for Esplora or mempoolspace
+    try:
+        # MutinyNet API
+        feerate_url = "{}/v1/fees/recommended".format(plugin.api_endpoint)
```

Comment:
I would swap the order of mutinynet and blockstream, hitting blockstream first, since that will be the default for most users.